jobserver/models.py

```diff
@@ -602,6 +628,8 @@ def get_remove_url(self):
 
 
 class Release(models.Model):
+    """Outputs released from a Workspace"""
```

Comment:
@bloodearnest – would love a quick check on this from you since you built the releases bits!

Reply:
Maybe "Review and redacted outputs from a Workspace"?

Reply:
Did you mean "Reviewed" here?

Reply:
Assuming yes and updated commit with better words, thanks!

Reply:
:facepalm: